src/Pages/History/History.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './History.css';
+import { SideBar , CategoryBar } from '../../components/index';
+import { Link } from 'react-router-dom';
+import { removeHistoryVideo , removeAllHistory  } from '../../api-calls/api-calls';
+import { useHistoryContext } from '../../context/HistoryContext';
+
+export function History() {
+    const { historyVideosState , historyVideosDispatch } = useHistoryContext();
+    let historyVideos;
```

Comment:
you should use useState for historyVideos instead of  let variable.

Reply:
ok, I will work on that.